website/frontend/templates/docs/tags.html

```diff
@@ -105,21 +105,33 @@ <h1 id="basic_tags">Basic Tags</h1>
           <td>date</td>
           <td> <a href="http://musicbrainz.org/doc/Release_Date" title="Release Date">Release Date</a> (YYYY-MM-DD)</td>
         </tr>
+        <tr>
+          <td>originalartist</td>
+          <td>Original <a href="http://musicbrainz.org/doc/Track_Artist" title="Track Artist">Track Artist</a> of the earliest release in the <a href="http://musicbrainz.org/doc/Release_Group" title="Release Group">Release Group</a> intended for the performer(s) of the original recording.</td>
+        </tr>
+        <tr>
+          <td>originalalbum</td>
+          <td>Original <a href="http://musicbrainz.org/doc/Release#Title" title="Release">Release Title</a> of the earliest release in the <a href="http://musicbrainz.org/doc/Release_Group" title="Release Group">Release Group</a> intended for the title of the original recording (or source of sound)</td>
+        </tr>
         <tr>
           <td>originaldate</td>
           <td>
-            Original <a href="http://musicbrainz.org/doc/Release_Date" title="Release Date">Release Date</a>
-            (YYYY-MM-DD) of the earliest release in the
-            <a href="http://musicbrainz.org/doc/Release_Group" title="Release Group">Release Group</a> intended
-            to provide e.g. the release date of the vinyl version of what you have on CD. (Included as standard
-            from Picard 0.15, and using the Original Release Date plugin if you are <b>still</b> using a non-NGS
-            version earlier than Picard 0.15)
-            <p>
-              Note: If you are storing tags in mp3 files as ID3v23 (which is the Windows / iTunes compatible
-              version) then original date can only be stored as a year.
-            </p>
+              Original <a href="http://musicbrainz.org/doc/Release_Date" title="Release Date">Release Date</a>
+              (YYYY-MM-DD) of the earliest release in the
+              <a href="http://musicbrainz.org/doc/Release_Group" title="Release Group">Release Group</a> intended
+              to provide e.g. the release date of the vinyl version of what you have on CD. (Included as standard
+              from Picard 0.15, and using the Original Release Date plugin if you are <b>still</b> using a non-NGS
+              version earlier than Picard 0.15)
+              <p>
+                Note: If you are storing tags in mp3 files as ID3v23 (which is the Windows / iTunes compatible
+                version) then original date can only be stored as a year.
+              </p>
           </td>
         </tr>
+        <tr>
+          <td>originalyear</td>
+          <td>Year of the Original <a href="http://musicbrainz.org/doc/Release_Date" title="Release Date">Release Date</a> intended for release year of the original recording.</td>
```

Comment:
This sentence is a bit confusing to me. «Year of the Original Release Date intended for release year of the original recording.» Is it referring to [Recording](https://musicbrainz.org/doc/Recording)? If so, "recording" should probably be capitalised like entities are elsewhere here. (And maybe also link to that /doc page.)

Maybe «Year of the original [Release Date] intended for the year the [Recording] was first released.»? (Note lowercase "original"—the other cases with uppercase "original" seem to be because "original" is the first word in a sentence, not because the word itself is "proper noun-like".)